docs/install/installing-istio.md

```diff
@@ -29,235 +29,180 @@ instructions. If you're familiar with Istio and know what kind of installation
+                  value:
+                    cluster-local-gateway
+                - path: spec.template.metadata.labels.app
+                  value:
+                    cluster-local-gateway
+            # Although we don't need this Gateway, we cannot remove it. Add "dummy" selector for the workaround.
+            # https://github.com/istio/istio/issues/22095
+            - kind: Gateway
+              name: cluster-local-gateway
+              patches:
+                - path: spec.selector.istio
+                  value:
+                    dummy
+EOF
+
+istioctl manifest apply -f istio-minimal-operator.yaml
 ```
 
 #### Installing Istio with sidecar injection
 
 If you want to enable the Istio service mesh, you must enable [automatic sidecar
-injection][1]. The Istio service mesh provides a few benefits:
+injection][2]. The Istio service mesh provides a few benefits:
 
-- Allows you to turn on [mutual TLS][4], which secures service-to-service
+- Allows you to turn on [mutual TLS][3], which secures service-to-service
   traffic within the cluster.
 
-- Allows you to use the [Istio authorization policy][5], controlling the access
+- Allows you to use the [Istio authorization policy][4], controlling the access
   to each Knative service based on Istio service roles.
 
-Enter the following command to install Istio:
-
-```shell
-# A template with sidecar injection enabled.
-helm template --namespace=istio-system \
-  --set sidecarInjectorWebhook.enabled=true \
-  --set sidecarInjectorWebhook.enableNamespacesByDefault=true \
-  --set global.proxy.autoInject=disabled \
-  --set global.disablePolicyChecks=true \
-  --set prometheus.enabled=false \
-  `# Disable mixer prometheus adapter to remove istio default metrics.` \
-  --set mixer.adapters.prometheus.enabled=false \
-  `# Disable mixer policy check, since in our template we set no policy.` \
-  --set global.disablePolicyChecks=true \
-  --set gateways.istio-ingressgateway.autoscaleMin=1 \
-  --set gateways.istio-ingressgateway.autoscaleMax=2 \
-  --set gateways.istio-ingressgateway.resources.requests.cpu=500m \
-  --set gateways.istio-ingressgateway.resources.requests.memory=256Mi \
-  `# More pilot replicas for better scale` \
-  --set pilot.autoscaleMin=2 \
-  `# Set pilot trace sampling to 100%` \
-  --set pilot.traceSampling=100 \
-  install/kubernetes/helm/istio \
-  > ./istio.yaml
-
-kubectl apply -f istio.yaml
-```
-
-#### Installing Istio with SDS to secure the ingress gateway
-
-Install Istio with [Secret Discovery Service (SDS)][3] to enable a few
-additional configurations for the gateway TLS. This will allow you to:
-
-- Dynamically update the gateway TLS with multiple TLS certificates to terminate
-  TLS connections.
-
-- Use [Auto TLS](../serving/using-auto-tls.md).
-
-The below `helm` flag is needed in your `helm` command to enable `SDS`:
+To automatic sidecar injection, set `autoInject: enabled` in addition to above
```

Comment:
I thought this would be `autoInjection.enabled=true` (or false)

Reply:
If we use the command line option, yes it is. But as we used the IstioOperator YAML above, I wanted to say "please set `autoInject: enabled` here https://github.com/knative/docs/pull/2458/files#diff-1116bcf887cf6b5e21e5e4f96fa026b1R59"